deploy/scripts/src/mobile/utils/config.ts

```diff
@@ -0,0 +1,5 @@
+export const config = {
```

Comment:
Would it help to add comments indicating what the __ENV means?

Reply:
Have added a comment here - do you think this makes it clearer?

Reply:
I was thinking of just saying something __ENV is the k6 shorthand for accessing environment variables. See here deploy/scripts/README.md for guidance on setting these variables?

Reply:
Didn't notice it was already well-documented there. Have updated!